src/notation/view/notationcontextmenumodel.cpp

```diff
@@ -200,6 +202,40 @@ MenuItemList NotationContextMenuModel::makeElementItems()
         items << makeMenuItem("edit-element");
     }
 
+    items << makeSeparator();
+
+    MenuItem* item = new MenuItem(this);
+
+    mu::ui::UiAction action;
+    action.code = "edit-style";
+    action.title = TranslatableString("notation", "Style...");
+    item->setAction(action);
+
+    QString id = QString::fromStdString(item->action().code) + QString::number(0);
+    item->setId(id);
+
+    mu::ui::UiActionState state;
+    state.enabled = true;
+    item->setState(state);
+
+    auto element = selection()->element();
+
+    if (element) {
+        if (element->isText()) {
+            // This directs to the tablature styles page for some reason...
+            item->setArgs(mu::actions::ActionData::make_arg2<QString, QString>("text-styles", "title"));
```

Comment:
You should use EditStyle::pageCodeForElement here

Reply:
Yep, this is currently a placeholder while I figure out why these arguments send me to tablature styles.